ansible/group_vars/omero-hosts.yml

```diff
@@ -163,15 +163,18 @@ omero_web_config_set:
   omero.web.public.cache.timeout: 60
   omero.web.page_size: 500
 
-
+  # NB: See additional CORS config below
+  omero.web.cors_origin_allow_all: True
```

Comment:
Minor point, but since the rest of the CORS config is in `omero_web_apps_*` I think this should go under [`omero_web_apps_config_set:`](https://github.com/IDR/deployment/pull/133/files#diff-564d3b63a1550c7a5968bd56c4adb048R266) instead

Reply:
Done